### PR TITLE
Generate sample leads given a list of fields

### PR DESCRIPTION
**Pull request by alexkwolfe**
The new `getSampleLead` function relies on the latest minor revision of leadconduit-types to generate sample data based on the type of each field. 

@cgrayson, if you please.

### CODE REVIEW

**Comment by justinwinslow** [comment]
Is this something the UI can use to allow users to submit test leads? Is that the intent and I'm just out of the loop?

**Comment by alexkwolfe** [comment]
The intent is to use this for posting instructions.

File: spec/sample-lead-spec.coffee [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    sample = fields.getSampleLead([ fields.lookup('state') ], 'pl')
+
+    # The state type doesn't know how to parse anything other than a US state. The 'pl' locale
+    # doesn't have any state names or state codes that overlap with US states.
+    parsed = types.state.parse(sample.state)
+    assert.equal parsed.name, sample.state
+    assert.equal parsed.valueOf(), sample.state
+
+
+  it 'should prefer field example', ->
```
Comment (cgrayson) [inline spec/sample-lead-spec.coffee]: I _think_ this means that fields can have their own example data, and that if they do, it should be used in preference over random/faker data. But it took me a good long study to get there from here. Maybe a comment or more descriptive test name?

**Comment by cgrayson** [comment]
Otherwise LGTM

**Pull request merged**